third_party/move/move-vm/runtime/src/interpreter.rs

```diff
@@ -1056,20 +1063,11 @@ where
                 Ok(())
             },
             NativeResult::LoadModule { module_name } => {
```

Comment:
is this module name or module_id?

Reply:
This is a module id, had to do this because the field in enum is called like that🤦‍♂️ let me rename :)

Reply:
Will keep as is and rename separately, to keep PR small otherwise people are already scared of many changes 🤦‍♂️🤣